### DISCUSSION TITLE
Reanimated v3.9.0 android perfomance issue.

### DISCUSSION THREAD

**Discussion by ReactorCSA** [software-mansion/react-native-reanimated#5943] [Q&A]
I've updated to RN 0.74 & Reanimated v3.9.0.
I created an animation with 78 objects (cards) they shuffle with the only translate property. The previous release (3.8.0) was  just a bit laggy and it was okay because it was not a high-end android. But now when I updated to 3.9.0 it became almost 1.5x times worse :(
When I enable new architecture it becomes 2 times worse.
Maybe you know what is the issue? Thanks in advance!

**Comment by MatiPl01** [comment]
Hey!

Could you try upgrading reanimated to the latest release (`3.14.0`) and seeing if it helps? We are aware of the bug introduced in `3.9.0` that was resolved in the latest release.
